crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -91,7 +91,12 @@ def health_check
         unready = []
         # We are ignoring the ceph nodes, as they should already be in crowbar_upgrade state
         NodeObject.find("NOT roles:ceph-*").each do |node|
-          unready << node.name unless node.ready?
+          next if node.ready?
+          if node.state == "shutdown" && node.roles.include?("nova-compute-kvm")
```

Comment:
do we want this only for kvm?

Reply:
I think we support only kvm compute nodes during the upgrade ... but as this is ECP specific anyway - are there any different ones?

Reply:
True, ECP only has KVM.